slp2mp4/dolphinrunner.py

```diff
@@ -146,7 +149,8 @@ def run(self, slp_file, num_frames):
             while self.count_frames_completed() < num_frames:
 
                 if time.perf_counter() - start_timer > MAX_WAIT_SECONDS:
-                    raise RuntimeError("Timed out waiting for render")
+                    print("WARNING: Timed out waiting for render")
+                    break
```

Comment:
I noticed that on games that went to time sometimes this condition was met and it would stop all other games from being recorded. So I bumped up the time and made it so it only prints a warning, then breaks and kills dolphin like normal.